### DISCUSSION TITLE
[Help me configure please!!] I have 10 pcs

### DISCUSSION THREAD

**Discussion by daitvd1997** [swar/Swar-Chia-Plot-Manager#711] [Q&A]
First of all, I want to sorry for not speaking English very well.

Equipments:
SSD : M.2 NVMe SSD 1T *2
Mem: 32Gb
Cpu:  i7-10700 ( 8x16)
My setting global: 
  max_concurrent: 8
  max_for_phase_1: 4
  minimum_minutes_between_jobs: 5
My setting setting for each SSD:
    size: 32
    bitfield: true
    threads: 2
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 4
    max_concurrent_with_start_early: 7
    initial_delay_minutes: 0
    stagger_minutes: 35
    max_for_phase_1: 2
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5 ]

**Finaly I get 15 plot each pc of days. This seems low by international standards.**

**Comment by Meyer85** [comment]
Put a bigger delay between plots as you are not using a staging drive
